digital_asset_types/src/rpc/response.rs

```diff
@@ -30,6 +30,8 @@ pub struct AssetList {
     pub before: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub after: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub cursor: Option<String>,
```

Comment:
The PR description mentions "`cursor` behaves like the  `after` option", so I was wondering why it is necessary to add `cursor`. From a devx perspective, it would be better to a single way to achieve something in the API, so if `cursor` and `after` are the same, then we should keep one of them.

Reply:
Discussed on a call. The main reason is for simplicity / UX: a developer does not want to have to re-think what before/after means as they adjust the sort order. Its far easier to take a token/cursor and fire-n-forget. 

However, you can use before/after in combination with before/after if you wanted to. 
E.g. scope down a query to a range and then paginate throughout that range.